components/cas-inbound-authenticator/src/main/java/org/wso2/carbon/identity/sso/cas/request/CASLogoutRequest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.sso.cas.request;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.identity.application.authentication.framework.inbound.FrameworkClientException;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * CAS logout request class.
+ */
+public class CASLogoutRequest extends CASIdentityRequest {
+
+    private static final Log log = LogFactory.getLog(CASLogoutRequest.class);
+
+    public CASLogoutRequest(CASSpInitRequestBuilder builder) throws FrameworkClientException {
+
+        super(builder);
+    }
+
+    public static class CASSpInitRequestBuilder extends CASIdentityRequestBuilder {
```

Comment:
Shall we add a class description here as well?

Reply:
Done